### PR TITLE
Adding docker to monolog-logdna

### PR DESCRIPTION
**Pull request by dannorcott**
I suspect we could trim some bits from the php.ini - i just copied it from the site one. Any ideas @ziadoz ?

### CODE REVIEW

**Comment by ziadoz** [comment]
@fusionspim/devs I've brought this up-to-date.

File: docker/php-cli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM php:7.4.15-cli
```
Comment (ziadoz) [inline docker/php-cli/Dockerfile:1]: Pretty simple really. A standard Docker PHP CLI image with Composer (Git/Zip/7Zip) and the default development configuration enabled, which should be sufficient.

File: .github/workflows/build-and-push.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+name: "Build and Push"
+
+on:
+  push:
+    branches:
+      - develop
```
Comment (maxakropolis) [inline .github/workflows/build-and-push.yml]: Is `develop` correct?

Comment (ziadoz) [reply .github/workflows/build-and-push.yml]: @maxakropolis No, have corrected.

**Review by maxakropolis — APPROVED** [review]

**Pull request merged**